src/vm.rs

```diff
@@ -118,6 +119,7 @@ pub struct UhyveVm<VCpuType: VirtualCPU = VcpuDefault> {
 	pub virtio_device: Arc<Mutex<VirtioNetPciDevice>>,
 	#[allow(dead_code)] // gdb is not supported on macos
 	pub(super) gdb_port: Option<u16>,
+	pub file_map: Option<UhyveFileMap>,
```

Comment:
The `pub` part was mentioned IRL, but it remains necessary for the `hypercall::open` to be able to access it.

Reply:
Indeed, the design of the hypercalls is not great (yet :slightly_smiling_face: ). But for the moment, `pub (crate)` or `pub (super)` would suffice.

Reply:
https://github.com/n0toose/uhyve/commit/c39102c1247f847b22ff73d421625b4145609be3